db/migrate/20151101224210_generate_token_for_existing_users.rb

```diff
@@ -0,0 +1,7 @@
+class GenerateTokenForExistingUsers < ActiveRecord::Migration
+  def change
+    User.all.each do |user|
```

Comment:
`find_each` is better option than `each` because it pulls data in batches of thousands from database and then iterate over. However performing data migration in place of schema migrations is a bit risky. If you model name changes, then your migrations won't run from start to end. I recommend to learn more about custom rake tasks.

Homework: Read more about Custom Rake tasks - rework this to a custom rake task.